### ISSUE TITLE
loading example sphere64 fails on Win10 (similar to #1550?)

### ISSUE THREAD

**Issue by MarkUoLeeds** [NCAR/VAPOR#2284]
### Describe the bug
Hello,
I am following https://www.docs.vapor.ucar.edu/gettingDataIntoVapor.html for the sphere64 from raw and VAPOR "crashes" i.e. suddenly closes, after I open the sphere64.vcd using GUI file explorer.

Lenovo Thinkpad core i5-6200 ; 8GB RAM ; Windows 10 Enterprise 64-bit

I do not know where to  look for any log of the fail.
I am new to VAPOR (and to some extent Windows.)

Mark

**Helpful additional information**
(*Please click check boxes AFTER submitting ticket*)
- [x] Did VAPOR crash?
- [ ] Did you get wrong results?

**Impact**
- [x] High - User productivity significantly degraded
- [ ] Medium - User productivity partially degraded
- [ ] Low - User productivity trivially degraded

### To Reproduce
Steps to reproduce the behavior. For example:
1. Go to 'File -> Open VDC'
2. Click on 'sphere64.vdc'
3. See VAPOR suddenly close

*Additionally, please attach a session file to this ticket to easily reproduce (use File->Save Session).*

### Expected behavior
A rendering of the sphere64 example
### Attachments
* Can you reproduce the bug on our test data?
  * If so, which file?
  * If not, access to your data would be very helpful. If the data set is small, please zip and upload it as an attachment. If it is large, please provide instructions for how we can contact you to obtain this data.
* Please attach a session file to easily reproduce (use File->Save Session).
* Please attach any screenshots relevant to understanding the issue.

### Desktop
 - OS and version: 64bit Windows 10 Enterprise 1803 build 17134.1246
 - Version: [e.g. Vapor 3.1]  3.2.0

### Additional context
Add any other context about the problem here. Attach any (zipped) relevant files, such as data, configurations, session files, etc.

**Comment by StasJ** [comment]
I followed the instructions both fully and skipping the populate step on both the current master and the 3.2 release on my mac and am unable to reproduce the problem. You may be able to get some error logging by running vapor from powershell. To do this, start powershell (it is included in windows by default) and then run vapor by typing the full path to the application (e.g. `C:\Program Files\VAPOR\vapor.exe`).

**Comment by MarkUoLeeds** [comment]
Hello, I was able to start a powershell and run vapor.exe. I then used the load vdc to navigate to the sphere64.vdc object and load that. Unfortunately the program crashes within 10 seconds so no idea of what or why this fails.
I have no other "premade" data. I can provide you with the sphere64.vdc structure that was created as it is only a few kB
[ex_sphere.zip](https://github.com/NCAR/VAPOR/files/4528913/ex_sphere.zip)

. Apologies for lagged response as I only have a limited time each week to try this.

**Comment by StasJ** [comment]
Hi Mark, it's no problem. When vapor crashed was there any output in the powershell window or were there any crash logs that were generated (e.g. windows asking would you like to report this crash)? I tested the dataset you provided and it works correctly for me.

If we cannot get this resolved, I would recommend trying to run vapor on a Mac or Linux system.

**Comment by MarkUoLeeds** [comment]
Thank you for testing that I had created sphere64 correctly.
I went and tested it on my personal desktop system (Win10 i5 6600 16GB RAM ) where I had done the primary install before copying vapor installed folder to my Work laptop (Win10 i5 6200U 8GB RAM); behaved the same :/

I do not use Macs and my home Linux systems are all >12 years old.
There was a different issue when I had tried to install on work office desktop (Centos 7.5) but I cannot access that since 17th March.

Thanks for trying. It will have to wait. [I cannot even install VMs on this work laptop.] But I might try on my home desktop with virtualbox.

Mark


Mark Richardson, Technical Head of Centre for Environmental Modelling and Compution (CEMAC),
School of Earth and Environment, University of Leeds, LS2 9JT
www.cemac.leeds.ac.uk<http://www.cemac.leeds.ac.uk>
(Note, although there is a “g” in my email address I have no second name)
